README.md

```diff
@@ -234,3 +234,67 @@ distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
+
+## SSO Authentication
```

Comment:
Keep the README unchanged since this feature is not released yet. You can a `docs/user-guide/auth.md` and put the content there. We can help polish it later